Create README.md

ToastyStoemp opened the pull request:
added, how to git clone

Commit:
Create README.md

GlenDC commented on README.md:
```diff
@@ -0,0 +1,5 @@
+Slides to the platform dev course
+
+to clone this repo use:
+    `git clone https://github.com/NuclearCookie/PlatformDevSlides.git --recursive`
```
You should probably add `--origin upstream`, such that your fork (for contributions) can be the remote `origin`.

Commit:
Added upstream origin

The latest changes can now easily be synced to your own fork
Thanks to @GlenDC for the insight.
